packages/plugin-vue-jsx/package.json

```diff
@@ -1,29 +1,20 @@
 {
   "name": "@vitejs/plugin-vue-jsx",
   "version": "4.2.0",
-  "type": "commonjs",
+  "type": "module",
   "license": "MIT",
   "author": "Evan You",
   "files": [
     "dist"
   ],
-  "main": "./dist/index.cjs",
-  "module": "./dist/index.mjs",
-  "types": "./dist/index.d.ts",
-  "exports": {
-    ".": {
-      "import": "./dist/index.mjs",
-      "require": "./dist/index.cjs"
-    }
-  },
+  "exports": "./dist/index.mjs",
```

Comment:
Suggest to expose `package.json` as well.

Reply:
It wasn't exposed before this PR as well. What is the reason for it?

Reply:
No specific reason, but I thought it might be good for resolver performance (in Vite?).

Reply:
I've added `./package.json` to exports.